src/OpenTelemetry.Extensions.Propagators/README.md

```diff
@@ -0,0 +1,47 @@
+# Trace context propagators for OpenTelemetry .NET
+
+[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Extensions.Propagators.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.Propagators)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Extensions.Propagators.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.Propagators)
+
+The package provides context propagators (currently
```

Comment:
please add a link to spec : https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md